test/charms/main/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+"name": "main"
+"summary": "A charm used for testing the basic operation of the entrypoint code"
+"maintainer": "Dmitrii Shcherbakov <dmitrii.shcherbakov@canonical.com>"
+"description": |
+  A charm used for testing the basic operation of the entrypoint code
+"tags":
+- "misc"
+"series":
+- "bionic"
+- "cosmic"
+- "disco"
+provides:
+  db:
+    interface: db
+requires:
+  mon:
+    interface: monitoring
+peers:
+  ha:
+    interface: cluster
+"subordinate": !!bool "false"
```

Comment:
This file needs some polishing for consistency. Sometimes keys have quotes, sometimes they don't. Sometimes values have quoting, sometimes they don't. Sometimes tags, sometimes not. Sometimes indented, sometimes not.

Let's please use this format consistently:
```
foo:
<four spaces>bar: false
baz:
<four spaces>- one
<four spaces>- two
`